src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+
+import TabItem from "./TabItem";
+
+import { StyledTabs } from "./Tabs.styled";
+
+const Tabs = (props) => {
+  const initValue = props.children[1].props.children.props.children;
```

Comment:
Oj, okropne rozwiązanie :P Postaraj się to rozwiązać w inny sposób - zbyt dużo zależności.
Najlepiej będzie wskazać numer "dziecka" i z resztą nie powinno być problemu.